docs/application/native/guides/connectivity/overview.md

```diff
@@ -36,7 +40,7 @@ You can transfer files between 2 devices. You can manage the Media Transfer Prot
 You can allow the users to initialize the VPN device, and manage routing, DNS, and firewall features.
 
 - [IoT Connectivity](iotcon.md)
-You can use IoTivity features in Tizen. You can use the seamless device-to-device connectivity to address the needs of the Internet of Things (IoT) through the open source reference implementation of the OIC (Open Interconnect Consortium) standard specifications.
+In Tizen use IoT Activity feature and seamless device-to-device connectivity to address the needs of the Internet of Things (IoT) through the open source reference implementation of the OIC (Open Interconnect Consortium) standard specifications.
```

Comment:
Rephrase to: 

The IoT feature of Tizen, uses seamless device-to-device connectivity to connect/communicate through the open source reference implementation of the OIC (Open Interconnect Consortium) standard specifications.